authd.proto

```diff
@@ -41,11 +41,17 @@ message StringResponse {
   string msg = 1;
 }
 
+enum SessionMode {
+  UNDEFINED = 0;
```

Comment:
Why having UNDEFINED? I think it’s better, as this is a Go project that the 0 value is useful, and so, default to authd.

Reply:
I was wondering about the same thing. I looked at other enum examples in gRPC and it seemed to me as though having an undefined value is idiomatic in some ways.

Reply:
Yeah, in general in APIs, that’s the way, but here, we are from Go to Go, so I would say the idiomatic way is the Go one? (also, there is arguably a main mode, which is auth)

Reply:
Also, one of the protobuf strengths is the multi-language compatibility so, even though we are using this in a Go project, we should not restrict it to it. Wdyt?

Reply:
Ack, happy to give it a pass :)